smqtk_iqr/utils/web.py

```diff
@@ -1,13 +1,15 @@
 import time
-from typing import Dict, Any, Tuple, Iterable, Union
+from typing import Tuple, Iterable, Union, Sequence, Hashable, Mapping
 import flask
 import requests
 
 from smqtk_core.dict import merge_dict
 
 
 def make_response_json(
-    message: str, return_code: int = 200, **params: Dict[str, Any]
+    message: str, return_code: int = 200,
+    **params: Union[Hashable, Sequence[Hashable],
+                    Sequence[Sequence[Hashable]], Mapping]
```

Comment:
I imagine this could probably remain as `Any`, instead of the explicit `Union` given the purpose of this function.